src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -1543,6 +1543,22 @@ public function validateUrl($attribute, $value)
         return preg_match($pattern, $value) > 0;
     }
 
+    /**
+     * Validate that an attribute is a valid UUID.
+     *
+     * @param  string $attribute
+     * @param  mixed  $value
+     * @return bool
+     */
+    public function validateUuid($attribute, $value)
+    {
+        if (! is_string($value)) {
+            return false;
+        }
+
+        return preg_match('/^[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{12}$/D', $value);
```

Comment:
As the check is case insensitive it makes sense to use the `i` option.
Also you can use `\d` for digits.

```php
return preg_match('/^[\da-f]{8}-[\da-f]{4}-[\da-f]{4}-[\da-f]{4}-[\da-f]{12}$/iD', $value);
```